src/raw/client.rs

```diff
@@ -500,7 +500,40 @@ impl<PdC: PdClient> Client<PdC> {
     /// ```
     pub async fn scan(&self, range: impl Into<BoundRange>, limit: u32) -> Result<Vec<KvPair>> {
         debug!("invoking raw scan request");
-        self.scan_inner(range.into(), limit, false).await
+        self.scan_inner(range.into(), limit, false, false).await
+    }
+
+    // Create a new 'scan' request but scans in "reverse" direction.
+    ///
+    /// Once resolved this request will result in a `Vec` of key-value pairs that lies in the specified range.
+    ///
+    /// If the number of eligible key-value pairs are greater than `limit`,
+    /// only the first `limit` pairs are returned, ordered by the key.
+    ///
+    ///
+    /// Reverse Scan queries continuous kv pairs in range (endKey, startKey],
```

Comment:
The description is different from that in TiKV:

https://github.com/tikv/tikv/blob/55c572a564c2826498135e71dc2713594647c33b/src/storage/mod.rs#L2420-L2429.
Can me make it consistent? Also for the comments in tests.

Reply:
Good point. So the comments mentioned in above link doesn't match with actual code. If you see the [integration tests](https://github.com/tikv/tikv/blob/55c572a564c2826498135e71dc2713594647c33b/tests/integrations/storage/test_storage.rs#L355)  as well then that also works with assumption that endKey is by default excluded and startKey is included in reverse scan as well (just like normal scan operation but just in reverse direction). Let me know if I'm missing anything here.